ornate_frontend/src/index.js

```diff
@@ -2,16 +2,18 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import "./index.css";
-import Home from "./pages/Home/Home";
-import Navbar from "./components/Navbar/Navbar";
-
+import Discover from "./pages/Discover";
+import Home from "./pages/Home";
+import SignUp from "./pages/SignUp";
+import Creator from "./pages/Creator";
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <BrowserRouter>
     <Routes>
-      <Route path="/" element={<Home />}>
-        <Route path="navbar" element={<Navbar />} />
-      </Route>
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/SignUp" element={<SignUp />}></Route>
```

Comment:
According to React naming convection routes are generally in `snake_case`. Please try to change them in your later PR's. To know more about, check this out -> https://www.upbeatcode.com/react/react-naming-conventions/